platform/darwin/src/MGLMultiPoint.h

```diff
@@ -6,16 +6,24 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- The `MGLMultiPoint` class is an abstract superclass used to define shapes
- composed of multiple points. You should not create instances of this class
- directly. Instead, you should create instances of the `MGLPolyline` or
- `MGLPolygon` classes. However, you can use the method and properties of this
- class to access information about the specific points associated with the line
- or polygon.
+ The `MGLMultiPoint` class is used to define shapes composed of multiple points. 
+ You can use the method and properties of this class to access information about 
+ the specific points associated with a line or polygon.
```

Comment:
Mention that this class also doubles as a superclass of MGLPolyline and MGLPolygon, despite being a distinct geometry.